escher/image/inliner.c

```diff
@@ -104,6 +104,13 @@ int main(int argc, char * argv[]) {
   FILE * implementation = fopen(implementationPath, "w");
   generateImplementationFromImage(implementation, lowerSnakeCaseName, camelCaseName, width, height, rowPointers);
   fclose(implementation);
+  
+  for (int i=0; i<height; i++) {
+    free(rowPointers[i]);
+  }
+  free(rowPointers);
+  free(png);
+  free(info);
 
```

Comment:
`png` and `info` objects should be destroyed by `png_destroy_read_struct` of the libpng API

Reply:
Fixed with https://github.com/RedGl0w/epsilon/commit/1f80f93eb45a1b87b72c4b6d91f257d3f9b4652f